Changing over to new logo

daviddavis opened the pull request:
Fixes #38

**Note** Will update the date on the blog post before merging.

mhrivnak commented:
Let's consider rolling this out with the 2.12 release.

daviddavis commented:
@mhrivnak cool. works for me.

bmbouter commented on _posts/2017-01-19-new-pulp-logo.md:
```diff
@@ -0,0 +1,30 @@
+[a small one with just the graphic that represent Pulp](/images/pulp_logo.png)
+and [a larger one that features the word "pulp"](/images/pulp_logo_big.png).
+
+While the old Pulp logo featured very muted colors which may be hard to see for the
```
/the visually impaired/those who see color differently/

bmbouter commented on _posts/2017-01-19-new-pulp-logo.md:
```diff
@@ -0,0 +1,30 @@
+author: David Davis
+---
+
+![New Pulp Logo](/images/pulp_logo_big.png)
```
When I rendered this on the site it didn't show correctly for me. It just showed the unclickable text "New Pulp Logo".

daviddavis replied:
So I'm using an absolute path which should render properly once we publish this to pulpproject.org. I could use a relative path (`../../../images/pulp_logo_big.png`). It looks kinda ugly though. What do you think?

bmbouter replied:
Oh that makes sense. Let's leave as is thanks.

bmbouter commented:
Where did the .svg files go? At the tip of HEAD in pulp/pulpproject.org they are [here](https://github.com/pulp/pulpproject.org/tree/gh-pages/images/logo/originals).

daviddavis commented:
@bmbouter sorry my branch was out of date. I removed `/images/logo` and moved the svg files to `/images/svg/logo`. Let me know if there's a better place for them though.

daviddavis commented:
@mhrivnak when is the release date of 2.12? Do you want to do this maybe like a couple days before or a couple days after?
